triagebot.toml

```diff
@@ -417,12 +417,16 @@ trigger_files = [
     "src/ci",
     "src/tools/compiletest",
     "src/tools/cargotest",
-    "src/tools/tidy",
     "src/tools/remote-test-server",
     "src/tools/remote-test-client",
     "src/tools/tier-check"
```

Comment:
I think we can add `src/tools/rustdoc-gui-test` here.

Reply:
Good point, I forgor that existed

Reply:
Maybe "miropt-test-tools" and "suggest-tests"

Reply:
Looked at #135072, probably `rustdoc-gui-test` only for now.

Reply:
Added the ones you mentioned.